batch/batch/worker/worker.py

```diff
@@ -1717,21 +1723,22 @@ async def cleanup(self):
         log.info(f'{self}: cleaning up')
         try:
             await check_shell(f'xfs_quota -x -c "limit -p bsoft=0 bhard=0 {self.project_id}" /host')
-
```

Comment:
So the reason we can delete this is because there's no scratch directory for JVMJobs? I'm really confused about the storage story here. There's an xfsquota command existing above, but deleted in another place.